src/plugins/vis_type_table/public/components/table_vis_component.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState, useCallback, useMemo } from 'react';
+import { EuiDataGridProps, EuiDataGrid } from '@elastic/eui';
+
+import { IInterpreterRenderHandlers } from 'src/plugins/expressions';
+import { Table } from '../table_vis_response_handler';
+import { TableVisConfig } from '../types';
+
+interface TableVisComponentProps {
+  table: Table;
+  visConfig: TableVisConfig;
+  handlers: IInterpreterRenderHandlers;
+}
+
+export const TableVisComponent = ({ table, visConfig, handlers }: TableVisComponentProps) => {
+  const { rows, columns } = table;
+  const [pagination, setPagination] = useState({ pageIndex: 0, pageSize: 5 });
```

Comment:
Can be done in future commits, but these default values should be pulled out as constants where they're easy to find and update.

Reply:
will change this in the future. These default values are just for current hook up purpose.